src/engine/bank02.asm

```diff
@@ -426,117 +2184,154 @@ GetPointerToDeckCards: ; 9048 (2:5048)
 	pop af
 	ret
 
-Func_905a: ; 905a (2:505a)
+ResetCursorPosAndBlink: ; 905a (2:505a)
 	xor a
-	ld [wceaf], a
-	ld [wceb0], a
-	ld [wcea3], a
+	ld [wCursorDuelXPosition], a
+	ld [wCursorDuelYPosition], a
+	ld [wDuelCursorBlinkCounter], a
 	ret
 
-Func_9065: ; 9065 (2:5065)
+; handle player input in menu in Your Play Area
+; works out which cursor coordinate to go to
+; and sets carry flag if A or B are pressed
+; input
+; returns a =  $1 if A pressed
+; returns a = $ff if B pressed
+HandleDuelMenuInput_YourPlayArea: ; 9065 (2:5065)
```

Comment:
The trailing _YourPlayArea_ seems weird here, as this function handles the menu input for only the main Check menu with the four items. I think I would simply use ``HandleCheckMenuInput`` for this, what do you think?